blog/2023/04/julia-1.9-highlights.md

```diff
@@ -0,0 +1,376 @@
+
+To address this need, you can now designate a task as interactive when you [`Threads.@spawn`](https://docs.julialang.org/en/v1/base/multi-threading/#Base.Threads.@spawn)  it:
+
+```julia
+using Base.Threads
+@spawn :interactive f()
+```
+You can set the number of interactive threads available using the following command:
+
+```julia
+julia --threads 3,1
+```
+
+This command starts Julia with 3 "normal" threads and one interactive thread (in the interactive thread pool).
+
+For more information, refer to the [manual section on multi-threading](https://docs.julialang.org/en/v1.9-dev/manual/multi-threading/#man-threadpools). This feature was introduced in [#42302](https://github.com/JuliaLang/julia/pull/42302) .
+
+## REPL
+
+### Contextual module REPL
+
+*Rafael Forquet*
+
+
+In Julia, the REPL evaluates expressions within the `Main` module by default. Starting with version 1.9, you can now change this to any other module. Many introspection methods, such as `varinfo`, which previously defaulted to examining the `Main` module, will now default to the REPL's current contextual module.
+
+This feature can be particularly useful when developing a package, as you can set the package as the current contextual module. To change the module, simply enter the module name in the REPL and execute Meta+M (often Alt+M), or use the `REPL.activate` command.
+
+
+```julia
+julia> @__MODULE__ # Shows module where macro is expanded
+Main
+
+# Typing Base.Math and pressing Meta-m changes the context module
+(Base.Math) julia> @__MODULE__
+Base.Math
+
+(Base.Math) julia> varinfo()
+  name           size summary                                      
+  ––––––––––– ––––––– –––––––––––––––––––––––––––––––––––––––––––––
+  @evalpoly   0 bytes @evalpoly (macro with 1 method)              
+  Math                Module                                       
+  ^           0 bytes ^ (generic function with 68 methods)         
+  acos        0 bytes acos (generic function with 12 methods)      
+  acosd       0 bytes acosd (generic function with 1 method)       
+  acosh       0 bytes acosh (generic function with 12 methods)     
+  acot        0 bytes acot (generic function with 4 methods) 
+...
+```
+
+### Numbered prompt
+
+*Kristoffer Carlsson*
+
+Drawing heavily on inspiration from the IPython shell (and other notebook based systems like Mathematica), the Julia REPL can enable a "numbered prompt" that stores evaluated objects in the REPL for later use and keeps track of the number of expressions that have been evaluated.
+
+Being able to refer to an earlier evaluated object can be useful if one for exampe forgets to store the result of a long computation to a variable and then executes something else (so that `ans` gets overwritten).
```

Comment:
```suggestion
Being able to refer to an earlier evaluated object can be useful if, for example, one forgets to store the result of a long computation to a variable and then executes something else (so that `ans` gets overwritten).
```